Fix/examples save path

raph-luc opened the pull request:
Related to this issue: https://github.com/pyansys/pyfluent/issues/1393 (see my comment there as well)

A few other changes:
- In downloads.py the variable `resp` was unused, also not used anywhere else that I could find in pyansys, so I removed it
- Changed `string.replace` to `re.sub("...$")` to force it to remove only the extension, and not potential `.zip` in the middle of the file name (like `test.zip.h` or something of the sort). `re` is also used elsewhere in pyansys, so it is not a completely new import

Notes/thoughts:
- urllib.request.urlretrieve is ported from python2 and may become deprecated in the future (see here https://docs.python.org/3/library/urllib.request.html#urllib.request.urlretrieve)
- maybe add output into the console explaining what is going on? Like " downloading file..." or "file already exists"
- as discussed with @mkundu1 this does not solve the underlying behavior that Fluent will by default save output files based on where it read input files from, but I think this makes sense and can be expected by the user. At least now pyfluent users can specify where to download example files to (and so Fluent will save its output there as well in this basic example workflow).

mkundu1 reviewed (approved):
Can you please add the logging calls for Note 2?

raph-luc commented:
> Can you please add the logging calls for Note 2?

Will do

raph-luc commented:
Added info logging calls, and removed get_ext as it was only used to verify the extension of the file name/path, which I believe is best done with a simple `string.endswith()` to avoid getting stuck with potential `zip` in the middle of the file path. 
This is what the logging looks like right now, input:
![image](https://user-images.githubusercontent.com/132297401/236843036-bb27c837-a9c1-4096-87c6-df9ade8b9721.png)
Output:
![image](https://user-images.githubusercontent.com/132297401/236843048-7ccba785-dbb4-4232-b4f3-06e8b6885d73.png)
